packages/client/bin/cli.ts

```diff
@@ -58,7 +58,7 @@ const args: ClientOpts = yargs(hideBin(process.argv))
     choices: networks.map((n) => parseInt(n[0])),
     default: undefined,
   })
-  .option('syncMode', {
+  .option('sync', {
```

Comment:
I'm not really sure why this but with this change the `--dataDir` parameter no longer accepts relative paths (e.g. `--dataDir=../../datadir`) but only absolute ones.  I don't think that's necessarily a bad thing but we should probably note in the parameter help about "only accepts absolute paths" or something to that effect since `yargs` gives a weird error about "unknown parameter --dataDir, --datadir, --data-dir`